packages/dataviews/src/dataviews.js

```diff
@@ -92,7 +92,7 @@ export default function DataViews( {
 	);
 	return (
 		<div className="dataviews-wrapper">
-			<VStack spacing={ 3 } justify="flex-start">
+			<VStack spacing={ 0 } justify="flex-start">
```

Comment:
I'm thinking maybe we can remove this nested div. That is, it would look like this:

```javascript
return (
	<VStack
		className="dataviews-wrapper"
		spacing={ 0 }
		justify="flex-start"
	>
		<HStack />
		<ViewComponent />
		<Pagination />
	</VStack>
);
```